timestamp-api/src/main/java/com/palantir/timestamp/TimestampService.java

```diff
@@ -40,4 +40,20 @@
     @Path("fresh-timestamps")
     @Produces(MediaType.APPLICATION_JSON)
     TimestampRange getFreshTimestamps(@QueryParam("number") int numTimestampsRequested);
+
+    /**
+     * Fast forwards the timestamp to the specified one so that no one can be served fresh timestamps prior
+     * to it from now on.
+     *
+     * Sets the upper limit in the TimestampBoundStore as well as increases the minimum timestamp that can
+     * be allocated from this instantiation of the TimestampService moving forward.
```

Comment:
Just a note: these 2 lines were specific to the impl for persistent and should be removed?